pages/common/pandoc.md

```diff
@@ -1,7 +1,15 @@
 # pandoc
 
-> General markup converter.
+> Convert documents between various formats.
 
 - Convert file to pdf (the output format is automatically determined from the output file's extension):
 
 `pandoc {{input.md}} -o {{output.pdf}}`
+
+- Convert a file to a specific output format (useful for when the extension alone is ambiguous):
+
+`pandoc {{input.docx}} -t {{markdown_github}} -o {{output.md}}`
```

Comment:
Possibly.

    $ pandoc --version
    pandoc 1.16.0.2

I tried paginated through pandoc's [list of releases](https://github.com/jgm/pandoc/releases), but saw no mention of its removal -- on the contrary, it seem to have existed for as long as there have been descriptive release notes.